storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -7965,6 +7961,16 @@ int main(int argc, char **argv) {
       xb::error() << "datadir must be specified.";
       exit(EXIT_FAILURE);
     }
+
+    /* warning if target is not prepared */
+    read_metadata();
+    std::cout << metadata_type_str << " metadata_type_str " << std::endl;
```

Comment:
This is not at all the correct way after the new PXB logging mechanism ;)

Reply:
removed :)